msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationKotlinTest.kt

```diff
@@ -550,6 +553,117 @@ class NativeAuthPublicClientApplicationKotlinTest : PublicClientApplicationAbstr
         assertEquals(accessToken, accessTokenTwo)
     }
 
+    /**
+     * Test sign in, get access token. Compare to token from getAccount()
+     */
+    @Test
```

Comment:
I assume more tests are following, to test the wide range of use cases that we have established for this feature?